docs/release_notes/draft.md

```diff
@@ -0,0 +1,7 @@
+# Release X.Y.Z
```

Comment:
Can we rename this file to `0.14.0-draft.md`?

Reply:
Done. 👍